toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -249,3 +249,6 @@ msopenjdk-11-11.0.14.1+1-LTS-31207.aarch64.rpm
 pyproject-rpm-macros-1.0.0~rc1-2.cm2.noarch.rpm
 audit-3.0.6-6.cm2.aarch64.rpm
 audit-libs-3.0.6-6.cm2.aarch64.rpm
+libcap-ng-0.8.2-1.cm2.aarch64.rpm
+libcap-ng-debuginfo-0.8.2-1.cm2.aarch64.rpm
```

Comment:
The debuginfo subpackage shouldn't be in the chroot manifests- `update_manifests.sh` should take out all of the debuginfo subpackages (except for the openssl one- not sure why, but we have an exception for that). We generally want to keep the chroot manifests as minimal as possible, given that we need to unpack the tarball for every SRPM build.

Can you run the manifests through the `update-manifests.sh` script? That should fix this problem.